pkg/sysctl/main.go

```diff
@@ -65,7 +68,7 @@ func main() {
 			}
 			err = sysctl(line)
 			if err != nil {
-				log.Fatal(err)
+				log.Println(fmt.Errorf("WARN: %v", err))
```

Comment:
I think this should be `log.Warnln(err.Error())` or `log.Fatalln(err.Error())`, otherwise LGTM to me.

Reply:
Thanks, @w9n! I didn't see Warnln

Reply:
Actually, @w9n, I just looked again - I don't see ant options other than `Println`, `Fatalln`, or `Panicln`, as this isn't using one of the third party logging libraries. See https://golang.org/pkg/log/

Reply:
sorry, i looked at `log "github.com/sirupsen/logrus"` in my project, so your Println seems good in the standard pkg

Reply:
No problem. Thanks for the review!